src/components/headersEditor/HeadersSection.tsx

```diff
@@ -0,0 +1,22 @@
+import { Col, Row } from 'antd';
+import React, { useState } from 'react';
+
+import HeadersEditor from './HeadersEditor';
+import UrlPreview from './UrlPreview';
+
+export default function HeadersSection() {
+  const [headers, setHeaders] = useState<Record<string, string>>({});
+
+  return (
+    <>
+      <Row gutter={[16, 24]}>
+        <Col span={24}>
+          <HeadersEditor onChange={setHeaders} />
+        </Col>
+        <Col span={24}>
+          <UrlPreview endpoint="https://rickandmortyapi.com/api" headers={headers} />
```

Comment:
смущает, что у компонента зарадкожен `endpoint`, возможно стоит подумать над
```suggestion
          <UrlPreview endpoint="endpoint" headers={headers} />
```
```js
const [endpoint, setEndpoint] = useState('https://rickandmortyapi.com/api');
```
или что-то вроде того

Reply:
хотела подумать об этом когда объединим наши фичи и передавать сюда эндпоит урл, но 100% надо исправлять - исправила. когда ты положишь всё в общий компонент, через useState буду сохранять ввод из инпута с эндпоинт урлом и передавать его в этот урл-превью (если вообще нам нужен этот превью)